tests/app/main/test_briefs.py

```diff
@@ -676,11 +676,9 @@ def test_essential_requirements_evidence_escapes_brief_data(self):
         doc = html.fromstring(res.get_data(as_text=True))
         questions = doc.xpath(
             "//*[@class='question-heading']")
-        # if item is excaped correctly it will appear as text rather than an element
+
         assert questions[0].find("h1") is None
-        assert questions[0].text.strip() == '&lt;h1&gt;Essential one with xss&lt;/h1&gt;'
```

Comment:
Hang on, if they are appearing as literal '<h1>' then they *aren't* being escaped, this sounds like the test is finding a bug?